### PR TITLE
Added  codecov badge to readme.

### LINKED ISSUE #114: Add codecov badge in README
## Description
As a developer,
I need to see the coverage report in the README,
so that I can know about the coverage and create more issues to increase coverage.

## Mocks

[codecov](https://codecov.io/gh/anitab-org/open-source-programs-backend)

### Update [Required]
- [ ] Add codecov badge of coverage in README.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
0.5 hours

### PR DESCRIPTION
**Pull request by himanshu007-creator**
### Description

Fixes #114 

### Type of Change:

- Code
- Quality Assurance
- User Interface
- Outreach

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 [![Open Source Love](https://badges.frapsoft.com/os/v3/open-source.svg?v=103)](https://github.com/anitab-org/open-source-programs-backend/)
 [![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/anitab-org/open-source-programs-backend)
 ![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)
-
+[![codecov](https://codecov.io/gh/anitab-org/open-source-programs-backend/branch/develop/graph/badge.svg)](https://codecov.io/gh/anitab-org/open-source-programs-backend)
```
Comment (codesankalp) [inline README.md]: add a blank line after this so that the next line will be properly indented and spaced.

**Review by codesankalp — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (himanshu007-creator) [reply README.md]: ok

**Review by codesankalp — APPROVED** [review]
LGTM @himanshu007-creator :tada:.

**Review by ashwinijha6 — DISMISSED** [review]
Looks good to me @himanshu007-creator  👍

File: README.md [hunk 2]
```diff
@@ -6,6 +6,7 @@
 [![Open Source Love](https://badges.frapsoft.com/os/v3/open-source.svg?v=103)](https://github.com/anitab-org/open-source-programs-backend/)
 [![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/anitab-org/open-source-programs-backend)
 ![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)
+[![codecov](https://codecov.io/gh/anitab-org/open-source-programs-backend/branch/develop/graph/badge.svg)](https://codecov.io/gh/anitab-org/open-source-programs-backend)
```
Comment (codesankalp) [inline README.md]: @himanshu007-creator as the repo name has been changed to anitab-forms-backend please make the necessary changes.

**Review by codesankalp — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (himanshu007-creator) [reply README.md]: made the changes

Comment (ashwinijha6) [reply README.md]: Yes even I thought that @codesankalp

**Review by ashwinijha6 — APPROVED** [review]

**Review by codesankalp — APPROVED** [review]

**Review by isabelcosta — APPROVED** [review]
Awesome work @himanshu007-creator 👏🏾

**Pull request merged**